core/trino-main/src/main/java/io/trino/server/security/ServerSecurityModule.java

```diff
@@ -113,6 +113,13 @@ public static Module authenticatorModule(String name, Class<? extends Authentica
                 combine(module, authModule));
     }
 
+    public static List<String> authenticationTypes(SecurityConfig config)
```

Comment:
This is a Guice module, not a library. This method is only used inside this class. Someone else is going to change this later to be private, since it doesn't need to be public.